.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build_and_test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x]
```

Comment:
Do we need to run this on two different Node versions? What do we gain, seeing as we're not really testing anything in Node?

Reply:
I don't think so - this was just how the default workflow was set up so I went with it. I'll make some changes.